apps/root/src/services/tierService.ts

```diff
@@ -208,16 +208,6 @@ export default class TierService extends EventsManager<TierServiceData> {
         missing[AchievementKeys.SWAP_VOLUME] = requiredSwapVolume - currentSwapVolume;
       }
 
-      // DCA Swaps (only for Tier 0)
-      if (nextTier.level === 0) {
-        const requiredDCASwaps = 30;
-        const currentDCASwaps = totalAchievements[AchievementKeys.DCA_SWAPS] || 0;
-        details[AchievementKeys.DCA_SWAPS] = { current: currentDCASwaps, required: requiredDCASwaps };
-        if (currentDCASwaps < requiredDCASwaps) {
-          missing[AchievementKeys.DCA_SWAPS] = requiredDCASwaps - currentDCASwaps;
-        }
-      }
-
```

Comment:
Also should remove the 'nft ownership' check. Every nextTier === 0 is useless at this point